ui/app/AppLayouts/Wallet/popups/NetworkSelectPopup.qml

```diff
@@ -0,0 +1,83 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.3
+import QtGraphicalEffects 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import utils 1.0
+
+// TODO: replace with StatusModal
+Popup {
```

Comment:
we need to start using StatusModal, but I know from discussions that we won't be focusing on the UI a 100% so then we can do this later too.